ci/test/00_setup_env.sh

```diff
@@ -53,7 +53,7 @@ export RUN_FUZZ_TESTS=${RUN_FUZZ_TESTS:-false}
 export BOOST_TEST_RANDOM=${BOOST_TEST_RANDOM:-1}
 # See man 7 debconf
 export DEBIAN_FRONTEND=noninteractive
-export CCACHE_MAXSIZE=${CCACHE_MAXSIZE:-100M}
```

Comment:
This is sourced usually through `test_run_all.sh`, right?
What was the reason for such low values before?